doc/source/tutorial/optimize.rst

```diff
@@ -1467,52 +1472,47 @@ where
     \end{bmatrix}
     \end{equation*}
 
-The last inequality constraint :math:`x_i \geq 0` is minimum value constraint for all decision variables, which can be
-applied as the ``bounds`` argument of :func:`linprog`. As you can see the API doc :func:`linprog`, the default values of
-``bound`` argument is ``(0, None)`` that means all decision variables are non-negative. So, we don't need to do anything
-the last inequality constraint in this case.
+Lastly, let's consider the minimum and maximum inequality constraints for each decision variable, which is known as
+"box constraint". These constraints can be applied as the ``bounds`` argument of :func:`linprog`.
+As you can see the API doc :func:`linprog`, the default values of ``bounds`` argument is ``(0, None)`` that means
+all decision variables are non-negative. Using None indicates that there is no bound. So, we need to set each bound
+as ``bounds`` argument in this case.
```

Comment:
Lastly, let's consider the separate inequality constraints on each decision variable, which are known as
"box constraints" or "simple bounds". These constraints can be applied using the ``bounds`` argument of :func:`linprog`.
As noted in the :func:`linprog` documentation, the default value of ``bounds`` is ``(0, None)``, meaning that the 
lower bound on each decision variable is ``0``, and the upper bound on each decision variable is infinity:
all the decision variables are non-negative.  Our bounds are different, so we specify the lower and upper bound on each decision variable as a tuple and group these tuples into a list:
```
x0_bounds = (0, None)
x1_bounds = (0, 5.0)
x2_bounds = (None, 0.5)
x3_bounds = (-3.0, 0)
bounds = [x0_bounds, x1_bounds, x2_bounds, x3_bounds]
```

Reply:
I fixed.